Userland/Applications/Magnifier/main.cpp

```diff
@@ -111,6 +111,16 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
             magnifier->pause_capture(action.is_checked());
         });
 
+    auto lock_location_action = GUI::Action::create_checkable(
+        "&Lock Location", { Key_L }, [&](auto& action) {
+            magnifier->lock_location(action.is_checked());
+        });
+
+    auto always_on_top_action = GUI::Action::create_checkable(
+        "&Always on Top", [&](auto& action) {
+            window->set_always_on_top(action.is_checked());
+        });
```

Comment:
This action is never added to a menu, and is unrelated to locking. Is this here by mistake?

Reply:
yup